objectives/breadth-first-search/bfs.py

```diff
@@ -0,0 +1,17 @@
+from bokeh.plotting import figure, output_file, show
+
+# prepare some data
+x = [1, 2, 3, 4, 5]
+y = [6, 7, 2, 4, 5]
```

Comment:
You've got some good preparation here for BFS; just keep in mind how you set up your data before you start implementing your algorithm.